src/test/groovy/com/onesignal/androidsdk/MainTest.groovy

```diff
@@ -1295,11 +1295,11 @@ class MainTest extends Specification {
         when:
         def results = runGradleProject([
             compileLines: """
-                compile 'com.android.support:appcompat-v7:26.0.0'
-                compile 'com.google.firebase:firebase-messaging:15.0.2'
-                compile 'com.google.firebase:firebase-core:16.0.0'
+    implementation 'com.google.android.gms:play-services-base:15.0.1'
```

Comment:
is this indentation ok?

Reply:
ops, didn't mean to commit this at all actually. Was just using this to test something.
Created fixup commit de91e2e59f788b4e39857a4f062759178c74849a to undo this change.

Reply:
Ops messed up one line in that fixup commit so just force push that fixup as fe5ae0792c7826394a42e77a2c4af76a6b15d512.
Sorry done messing with it, should be good now